### ISSUE TITLE
Show Non-Dismissible Alert if `currentActivity` is `null` on App Start

### ISSUE THREAD

**Issue by Parveshdhull** [status-im/status-react#21585]
## Description

If `currentActivity` is `null` when the app starts, it leads to an unusable state where users can't access accounts or create new ones. To prevent users from being stuck in this state, we should display a native, non-dismissible alert. The alert should inform the user that the app started in an unusual state and prompt them to force close and restart the app.
more details: https://github.com/status-im/status-mobile/issues/20930#issuecomment-2454491957 https://github.com/status-im/status-mobile/pull/21553#issuecomment-2456518549

**Expected Behavior**:  
- Show a non-dismissible native alert if `currentActivity` is `null` during app startup.
- The alert should instruct the user to force close and restart the app.

**Comment by Parveshdhull** [comment]
From https://github.com/status-im/status-mobile/issues/20930#issuecomment-2454491957

> I created a pull request https://github.com/status-im/status-mobile/pull/21553 to properly initialize the push notification service, and I hope this will resolve the issue.

> If this occurs again, instead of alarming the user with a setup screen, we should display a non-dismissible popup informing them that the app has started in an unusual state and prompting them to force close and restart the app.

We haven't seen this bug in a long time, so I believe that https://github.com/status-im/status-mobile/issues/21553 has fixed it for good, and we can skip implementing this alert. @ilmotta

**Comment by ilmotta** [comment]
> We haven't seen this bug in a long time, so I believe that [#21553](https://github.com/status-im/status-mobile/pull/21553) has fixed it for good, and we can skip implementing this alert. [@ilmotta](https://github.com/ilmotta)

Good point @Parveshdhull, so we can close this issue indeed. A nice outcome after all 💯
